tutorials/polkadot-sdk/build-a-blockchain/upgrade-a-running-network.md

```diff
@@ -0,0 +1,262 @@
+    ...
+    ```
+
+2. Locate the `#[frame_support::runtime]` macro and add the Utility pallet:
+
+    ```rust
+    // Create the runtime by composing the FRAME pallets that were previously configured.
+    #[frame_support::runtime]
+    mod runtime {
+        ...
+        // Include the custom logic from the pallet-template in the runtime.
+        #[runtime::pallet_index(7)]
+        pub type TemplateModule = pallet_template;
+
+        #[runtime::pallet_index(8)]
+        pub type Utility = pallet_utility;
+        ...
+    }
+    ```
+
+#### Update Existential Deposit Value
+
+To update the `EXISTENTIAL_DEPOSIT` in the Balances pallet, locate the constant and set the value to `1000`:
+
+```rust
+...
+/// Existential deposit.
+pub const EXISTENTIAL_DEPOSIT: u128 = 1000;
+...
+```
+
+!!!note
+    This change increases the minimum balance required for accounts to remain active. No accounts with balances between `500` and `1000` will be removed. For account removal, a storage migration is needed. See [Storage Migration]() for details.
+
+#### Update Runtime Version
+
+Locate the `runtime_version` macro and increment the `spec_version` field from `100` to `101`:
+
+```rust
+#[sp_version::runtime_version]
+pub const VERSION: RuntimeVersion = RuntimeVersion {
+	spec_name: create_runtime_str!("solochain-template-runtime"),
+	impl_name: create_runtime_str!("solochain-template-runtime"),
+	authoring_version: 1,
+	spec_version: 101,
+	impl_version: 1,
+	apis: RUNTIME_API_VERSIONS,
+	transaction_version: 1,
+	state_version: 1,
+};
+```
+
+### Recompile the Runtime
+
+Once you've made all the necessary changes, recompile the runtime by running:
+
+```bash
+cargo build --release
+```
+
+The build artifacts will be output to the `target/release` directory. The Wasm build artifacts can be found in the `target/release/wbuild/solochain-template-runtime` directory. You should see the following files:
+
+- *solochain_template_runtime.compact.compressed.wasm*
+- *solochain_template_runtime.compact.wasm*
+- *solochain_template_runtime.wasm*
+
+## Execute the Runtime Upgrade
+
+Now that you've generated the Wasm artifact for your modified runtime, it's time to upgrade the running network. This process involves submitting a transaction to load the new runtime logic.
+
+### Understand Runtime Upgrades
+
+#### Authorization with Sudo
+
+In production networks, runtime upgrades typically require community approval through governance. For this tutorial, the Sudo pallet will be used to simplify the process. The Sudo pallet allows a designated account (usually `Alice` in development environments) to perform privileged operations, including runtime upgrades.
+
+#### Resource Accounting
+
+Runtime upgrades use the `set_code` extrinsic, which is designed to consume an entire block's resources. This design prevents other transactions from executing on different runtime versions within the same block. The `set_code` extrinsic is classified as an Operational call, meaning it:
```

Comment:
What is an "Operational" call? Not familiar with this term

Reply:
I'm not sure. I searched for documentation, but I couldn't find anything. It is written that way in the [Resource accounting for runtime upgrades](https://docs.substrate.io/tutorials/build-a-blockchain/upgrade-a-running-network/#authorize-an-upgrade-with-sudo) section of substrate docs. I quote:

"The weight annotation for the set_code function also specifies that the function is in the Operational class because it provides network capabilities. Function calls that are identified as operational"

Reply:
OK looked a little more, it is indeed a thing, see: 

- https://paritytech.github.io/polkadot-sdk/master/frame_support/dispatch/enum.DispatchClass.html#
- An example: https://github.com/paritytech/polkadot-sdk/blob/c8d5e5a383c01ee02c3cc49fbd5e07540b6b79cc/substrate/frame/system/src/limits.rs#L329

Reply:
For now, this is fine, we can just merge it as-is I'm pretty sure.

Reply:
I think adding a link to the polkadot-sdk docs would be helpful here